src/main/java/uk/gov/ons/ctp/response/lib/common/time/TimeHelper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ons.ctp.response.lib.common.time;
+
+import java.sql.Timestamp;
+
+/**
+ * This interface is to provide a layer of separation
+ * between the service classes and DateTimeUtil.
+ *
+ * This will aid us in incrementally removing the
+ * XMLGregorianCalendar from the services.
+ */
+public interface TimeHelper {
```

Comment:
I don't think this is a useful abstraction; we can just remove instances of `DateTimeUtil.giveMeCalendarForNow();` and replace them with `DateTimeUtil.nowUTC();`, I don't think we're ever going to make a different implementation of `TimeHelper` that we switch over to

Reply:
I have removed the TimeHelper interface for now to keep things simple. However, I disagree that we will not make different implementations as that is exactly what we should be doing for integration testing.